spec/facades/photo_facade_spec.rb

```diff
@@ -2,42 +2,20 @@
 
 RSpec.describe 'Photo Facade' do
   describe 'Photo', :vcr do
-    it 'makes an API call using the Unsplash gem' do
+    it 'makes an API call using the Unsplash gem and returns a photo with credits' do
       response = PhotoFacade.photos("cat")
       response_json = response.to_json
```

Comment:
I am not sure what this line is doing. 

Doesn't L4 in the PhotoFacade already do this? 
I am curios to know what `response` is (format).

Reply:
Updated, changes pushed up